cmd/discord-provider/server/operations.go

```diff
@@ -16,8 +21,32 @@ func NewServer() *Server {
 }
 
 func (server *Server) SendMessage(ctx context.Context, message *discord_provider.SendMessageRequest) (*discord_provider.SendMessageResponse, error) {
-	// Sends a message to a Discord webhook here
+	url, found := os.LookupEnv("DISCORD_WEBHOOK_URL")
+	if !found {
+		return &discord_provider.SendMessageResponse{}, fmt.Errorf("the env var DISCORD_WEBHOOK_URL is not set to a value")
+	}
+
+	discordName := fmt.Sprintf("**Name:** %s %s", message.Sender.FirstName, message.Sender.LastName)
+	discordEmail := fmt.Sprintf("**Email:** %s", message.Sender.Email)
+	discordPhone := fmt.Sprintf("**Phone:** %s", message.Sender.PhoneNumber)
+	discordRequestDetails := fmt.Sprintf("**Request details:** %s", message.RequestDetails)
+	discordRequestServices := fmt.Sprintf("**Request services:** %s", message.RequestService)
+	discordMessage := Message{
+		Username: "calidum-rotae",
+		Embeddeds: []Embedded{{Title: "New submission",
+			Description: fmt.Sprintf("%s\n%s\n%s\n\n%s\n%s", discordName, discordEmail, discordPhone, discordRequestDetails, discordRequestServices),
+			Color:       "16745728", // orange
+			Footer: Footer{
+				Text: "By calidum-rotae services",
+			}}},
+	}
+
+	payload := new(bytes.Buffer)
+	json.NewEncoder(payload).Encode(discordMessage)
```

Comment:
Does `.Encode` return something such as an error? If so we should handle it

Reply:
Yes, I will handle it